admin-site/src/App.svelte

```diff
@@ -0,0 +1,421 @@
+		document.getElementById("confirmDelete").style.display = "none";
+		document.getElementById("closeEvent").style.display = "block";
+	}
+
+	function deleteOccasion(){
+		// deletion code goes here
+		document.getElementById("confirmDelete").style.display = "none";
+		document.getElementById("eventsList").style.display = "block";
+		//removes occasion from object and from list
+		focusedEvent.occasions.splice(indexInOccasions,1);
+		document.getElementById(focusedOccasionID).remove();
+
+	}
+	function backToEvents(){
+		let id =this.value;
+		document.getElementById(id).style.display = "none";
+		document.getElementById("eventsList").style.display = "block";
+	}
+	function cancelEnd(){
+		
+		document.getElementById("confirmEndOccasion").style.display = "none";
+		document.getElementById("openEvent").style.display = "block";
+	}
+	function confirmEnd(){
+		
+		document.getElementById("openEvent").style.display = "none";
+		document.getElementById("confirmEndOccasion").style.display = "block";
+	}
+
+	function backToOccasionList() {
+		let id = this.value;
+		document.getElementById(id).style.display = "none";
+		document.getElementById("occasionList").style.display = "block";
+		
+	}
+	function showQR(){
+		let id = this.value;
+		document.getElementById(id).style.display = "none";
+		document.getElementById("QRcode").style.display = "block";
+
+	}
+
+//this changes which cue details are shown
+	function changeCueState(){
+		let direction = this.value;
+		if(direction == "next" && cueState <= focusedEvent.cues.length-1){
+			cueState += 1;
+		} else if(direction == "previous" && cueState > 1){
+			cueState -= 1;
+		} else {
+			cueState = 1;
+		}	
+	}
+
+/////for new event generation
+	function setTitle(event){
+		label = event.target.value;
+	}
+
+//whichever parameters we want to be able to build for new events from the site would go below
+	function createEvent(){
+		let newEvent = [{
+				label:label
+			}];
+		events = events.concat(newEvent);
+	}
+////////
+</script>
+
+<style>
+	#eventsList, #closeEvent,#openEvent, #occasionList, #QRcode, #confirmDelete, #confirmEndOccasion {
+		display: none;
+	}
+	
+	
+</style>
+
+
+<!-- Keeping this as a component cause it will likely need switching out -->
+<section id = "login">
+	<Login />
+
+</section>
+
+<!-- #eventsList allows a list of events to be built and shown -->
+
+<section id = "eventsList">
+	<h1 class="text-center">Events</h1>
+	<hr>
+		{#if events.length === 0}
+			<p>No events have been added yet</p>
+			{:else}
+					{#each events as event}
+						<button type="button" class= 'btn btn-primary btn-block' value = {event.id} on:click={eventButton} >
+							<h3>{event.label}</h3>	
+						</button>
+					{/each}
+		{/if}
+		<hr>
+		<!-- event creation -->
+		<div>
+			<label for="title">New Event Name</label> 
+			<input type="text" id="title" value={label} on:input={setTitle}>  
+		</div>
+		<button class = "btn btn-primary" on:click={createEvent}>Create New Event</button>
```

Comment:
Let's adjust this a bit, can you try using an [input group with button addon](https://getbootstrap.com/docs/4.3/components/input-group/#button-addons) to get the controls to one line?

**Add new event**
[  text field, placeholder: 'Event name'][ Add event ]

Reply:
Change applied